net/unbound/files/unbound.sh

```diff
@@ -1067,28 +1065,33 @@ unbound_start() {
 
     unbound_control
   fi
+
+  if unbound_ismain ; then
+    # set resolver file to local.
+    rm -f /tmp/resolv.conf
+
+    {
+      echo "nameserver 127.0.0.1"
+      echo "nameserver ::1"
+      echo "search $UNBOUND_TXT_DOMAIN"
+    } > /tmp/resolv.conf
+  fi
 }
 
 ##############################################################################
 
 unbound_stop() {
-  local resolvsym=0
-
-  rootzone_update
-
-
-  if [ ! -x /usr/sbin/dnsmasq -o ! -x /etc/init.d/dnsmasq ] ; then
-    resolvsym=1
-  else
-    /etc/init.d/dnsmasq enabled || resolvsym=1
-  fi
+  config_load unbound
+  config_foreach unbound_uci unbound
```

Comment:
I dont like spooling the whole UCI for the tear down event. Also if someone is testing dnsmasq and unbound back to back by flipping enable/disable, then there will be racing consequences. If Unbound installs at 53 but is UCI change-reload to 5353, then this also. Perhaps consider leaving a comment in resolv.conf that is `case $( cat ) in ))) esac`. Concept "I did this, so I will fix it."